src/System.Private.Uri/tests/FunctionalTests/UriBuilderTests.cs

```diff
@@ -289,9 +290,23 @@ public void Query_Get_Set(string value, string expected)
         }
 
         [Theory]
-        [InlineData("fragment", "#fragment")]
         [InlineData("#fragment", "#fragment")]
         [InlineData("#", "#")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "See: dotnet/corefx #15145")]
+        public void Fragment_Get_Set_Netcore(string value, string expected)
```

Comment:
Rather than a "_NetCore" suffix, it'd be better to describe the behavior being tested.